examples/3dtiles.html

```diff
@@ -0,0 +1,119 @@
+
+            div {
+                margin : auto auto;
+                width: 100%;
+                padding: 0;
+                height: 100%
+                /*margin-top: 50vh;
+                transform: translateY(-50%);*/
+            }
+            #menuDiv {position: absolute; top:0px; margin-left: 0px;}
+
+
+        </style>
+        <meta charset="UTF-8">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="/node_modules/dat.gui/build/dat.gui.min.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv"></div>
+        <div id="viewerDiv2"></div>
+        <script src="/examples/GUI/GuiTools.js"></script>
+        <script src="/dist/itowns.js"></script>
+        <script src="/dist/debug.js"></script>
+        <script type="text/javascript">
+            /* global itowns,document,GuiTools*/
+
+            const positionOnGlobe = { longitude: -75.539659, latitude: 39.539659, altitude: 100000 };
+
+            // iTowns namespace defined here
+            const viewerDiv = document.getElementById('viewerDiv');
+
+            const apiGlobe = new itowns.ApiGlobe();
+            const menuGlobe = new GuiTools(apiGlobe, 'menuDiv');
+            const globe = apiGlobe.createSceneGlobe('globe', positionOnGlobe, viewerDiv);
+            // itowns.debug = new Debug(itowns.viewer.scene);
+
+            const urlImageryLayers = [];
+            const urlElevationLayers = [];
+
+            urlImageryLayers.push('layers/JSONLayers/Ortho.json');
+
+            urlElevationLayers.push('layers/JSONLayers/IGN_MNT.json');
+            urlElevationLayers.push('layers/JSONLayers/IGN_MNT_HIGHRES.json');
+
+            const imageryPromise = apiGlobe.addImageryLayersFromJSONArray(urlImageryLayers);
+            const elevationPromise = apiGlobe.addElevationLayersFromJSONArray(urlElevationLayers);
+
+            imageryPromise.then((layersColors) => { menuGlobe.addImageryLayersGUI(layersColors); });
+            elevationPromise.then((layersElevations) => { menuGlobe.addElevationLayersGUI(layersElevations); });
+
+            menuGlobe.addGUI('RealisticLighting', false,
+                (newValue) => { apiGlobe.setRealisticLightingOn(newValue); });
+
+            // add 3d-tiles layer
+            const threeDTilesLayer = new itowns.GeometryLayer('3d-tiles');
+
+            threeDTilesLayer.preUpdate = (context, layer) => {
+                if(layer.root === undefined) {
+                    itowns.init3DTilesLayer(context, layer);
+                    return [];
+                }
+                itowns.pre3DTilesUpdate(context, layer);
+                return [layer.root];
+            };
+            threeDTilesLayer.update = itowns.process3DTilesNode(
+                itowns.threeDTilesCulling,
+                itowns.threeDTilesSubdivisionControl
+            );
+            threeDTilesLayer.url = 'http://localhost:8003/tilesets/TilesetWithDiscreteLOD/tileset.json';
```

Comment:
Is your branch up-to-date? Did you open the right demo (localhost:8080/examples/3dtiles.html)?